ques1.rb

```diff
@@ -0,0 +1,18 @@
+puts "Enter a number:"
```

Comment:
Use single quotes rather than double quotes when there is no string interpolation.